frontend/components/SmileFooter/connector.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import { getSmileCustomer } from '../../selectors';
+
+/**
+ * @param {Object} state Redux state
+ * @return {Object}
+ */
+const mapStateToProps = state => ({
+  smileCustomer: getSmileCustomer(state),
```

Comment:
since the customer data is not really needed in the component we don't need to pass it. 
```suggestion
  smileCustomerAvailable: !!getSmileCustomer(state),
```

Reply:
Updated